zipline/finance/performance/tracker.py

```diff
@@ -355,7 +367,7 @@ def handle_minute_close(self, dt, data_portal):
         minute_packet = self.to_dict(emission_type='minute')
         return minute_packet
 
-    def handle_market_close(self, dt, data_portal):
+    def handle_market_close(self, dt, data_portal, calculate_position_weights):
```

Comment:
I'm wondering if there's a better way to flag whether or not to do this computation other than just passing it as an argument. Or, if it should just always be calculated.